ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -1502,41 +1534,43 @@ router_bgp:
 
 The change has been incorporated into `traffic-policies.j2` eos template.
 
-In AVD 4.0.0, configuring the `flags` for `UDP` protocol under traffic-policies was allowed and the configuration for `source/destination port` and `source/destination field-set` together was not generated for `TCP/UDP` protocol.
+In AVD 4.0.0, configuring the `flags` for the `UDP` protocol under traffic policies was allowed, but the configuration for `source/destination port` and `source/destination field-set` together was not generated for the `TCP/UDP` protocols.
 
-As of AVD 5.0.0, the configuration of `flags` in `UDP` protocol under traffic-policies is not generated and the configuration for `source/destination port` and `field set` can be generated together for `TCP/UDP` protocol. Also, the configuration order has been fixed as per the EOS running-config.
+As of AVD 5.0.0, the configuration of `flags` in the `UDP` protocol under traffic policies is not generated, and the configuration for `source/destination port` and `field set` can be generated together for the `TCP/UDP` protocol. Also, the configuration order has been fixed as per the EOS running-config.
```

Comment:
```suggestion
As of AVD 5.0.0, the configuration of `flags` in the `UDP` protocol under traffic policies is now generated, and the configuration for `source/destination port` and `field set` can be generated together for the `TCP/UDP` protocol. Also, the configuration order has been fixed as per the EOS running-config.
```

Reply:
It should be `not` only, earlier (AVD 4.x)  the `protocol udp flags established` command was generated which is invalid in EOS CLI. It has been fixed in this PR - https://github.com/aristanetworks/avd/pull/4266/

Reply:
thanks @Shivani-gslab I re-worded [46002ab](https://github.com/aristanetworks/avd/pull/4575/commits/46002ab6b043e6314d96e4b684fd7fd65ad129fe)